### PR TITLE
New package: SpaceTime.Sheas.Cealer version 1.1.1

### PR DESCRIPTION
**Pull request by SpaceTimee**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/159929)

### CODE REVIEW

**Comment by SpaceTimee** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-06-25T17:03:21.0425007Z Manifest Validation Failed
 2024-06-25T17:03:26.1395106Z ##[error]Manifest Validation Failed

(Automated response - build 886.)

**Comment by SpaceTimee** [comment]
There isn't any mistake on my machine

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 886.)

File: manifests/s/SpaceTime/Sheas/Cealer/1.1.1/SpaceTime.Sheas.Cealer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: SpaceTime.Sheas.Cealer
+PackageVersion: 1.1.1
+InstallerType: inno
+Installers:
```
Comment (stephengillie) [inline manifests/s/SpaceTime/Sheas/Cealer/1.1.1/SpaceTime.Sheas.Cealer.installer.yaml:7]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.8
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Manual Validation ended with:
image
image

File: manifests/s/SpaceTime/Sheas/Cealer/1.1.1/SpaceTime.Sheas.Cealer.locale.en.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+Publisher: Space Time
+PublisherUrl: https://www.spacetimee.xyz
+PrivacyUrl: https://thoughts.teambition.com/share/6264eda98adeb10041b92fda#title=Sheas_Cealer_%E9%9A%90%E7%A7%81%E6%94%BF%E7%AD%96
+Author: Space Time
+PackageName: Sheas Cealer
+PackageUrl: https://github.com/SpaceTimee/Sheas-Cealer
+License: No License
+Copyright: Copyright © 2077
+ShortDescription: Just Ceal It
+Description: Just Ceal It
```
Comment (Trenly) [inline manifests/s/SpaceTime/Sheas/Cealer/1.1.1/SpaceTime.Sheas.Cealer.locale.en.yaml:16]: Please provide a better description of what this application is, what it does, or what it is used for

File: manifests/s/SpaceTime/Sheas/Cealer/1.1.1/SpaceTime.Sheas.Cealer.locale.en.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+PackageIdentifier: SpaceTime.Sheas.Cealer
+PackageVersion: 1.1.1
+PackageLocale: en
+Publisher: Space Time
+PublisherUrl: https://www.spacetimee.xyz
+PrivacyUrl: https://thoughts.teambition.com/share/6264eda98adeb10041b92fda#title=Sheas_Cealer_%E9%9A%90%E7%A7%81%E6%94%BF%E7%AD%96
+Author: Space Time
+PackageName: Sheas Cealer
+PackageUrl: https://github.com/SpaceTimee/Sheas-Cealer
+License: No License
```
Comment (Trenly) [inline manifests/s/SpaceTime/Sheas/Cealer/1.1.1/SpaceTime.Sheas.Cealer.locale.en.yaml:13]:
```suggestion
License: Freeware
```

File: manifests/s/SpaceTime/Sheas/Cealer/1.1.1/SpaceTime.Sheas.Cealer.installer.yaml (hunk 1 above)
Comment (SpaceTimee) [reply manifests/s/SpaceTime/Sheas/Cealer/1.1.1/SpaceTime.Sheas.Cealer.installer.yaml:10]: Validation-Unattended-Failed still after this change

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> MSI (s) (54:B8) [04:10:42:858]: Executing op: ComponentRegister(ComponentId={C432B642-D1A7-53B2-814C-C3681BDC886D},KeyPath=22*85*\SOFTWARE\Microsoft\Windows\Windows Error Reporting\RuntimeExceptionHelperModules\C:\Program Files\dotnet\shared\Microsoft.NETCore.App\8.0.6\mscordaccore.dll,State=3,,Disk=1,SharedDllRefCount=0,BinaryType=1)
> MSI (s) (54:B8) [04:10:44:323]: Executing op: RegOpenKey(Root=-2147483646,Key=SOFTWARE\Microsoft\Windows\Windows Error Reporting\RuntimeExceptionHelperModules,,BinaryType=1,,)
> Property(S): ErrorDialog = ErrorDlg

(Automated response - build 886.)

**Pull request closed without merging**